packages/components/src/select-control/test/select-control.test.js

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
A convention in the codebase is that test file name should try to map to the filename that's being tested in the parent folder, so here it should be called `index.js` (and it doesn't need the `test.js` postfix).

Reply:
Thanks - done in 6c4e3a219e